src/modules/fancyzones/editor/FancyZonesEditor/Themes/HighContrastBlack.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:system="clr-namespace:System;assembly=System.Runtime">
+
+    <!--  Metadata  -->
+    <system:String x:Key="Theme.Name">HighContrast.Accent4</system:String>
+    <system:String x:Key="Theme.Origin">PowerToysRun</system:String>
+    <system:String x:Key="Theme.DisplayName">Accent4 (HighContrast)</system:String>
+    <system:String x:Key="Theme.BaseColorScheme">HighContrast</system:String>
+    <system:String x:Key="Theme.ColorScheme">Accent4</system:String>
+    <Color x:Key="Theme.PrimaryAccentColor">White</Color>
+
+    <SolidColorBrush x:Key="PrimaryBackgroundBrush" Color="#FF242424" />
+    <SolidColorBrush x:Key="SecondaryBackgroundBrush" Color="#FF1c1c1c" />
+    <SolidColorBrush x:Key="TertiaryBackgroundBrush" Color="#FF202020" />
+    <SolidColorBrush x:Key="PrimaryForegroundBrush" Color="#FFffffff" />
+    <SolidColorBrush x:Key="SecondaryForegroundBrush" Color="#FF1aebff" />
+    <SolidColorBrush x:Key="BackdropBrush" Color="#88000000" />
+    <SolidColorBrush x:Key="TitleBarSecondaryForegroundBrush" Color="#FF9a9a9a" />
+    <SolidColorBrush x:Key="LayoutItemBackgroundBrush" Color="#FF2b2b2b" />
+    <SolidColorBrush x:Key="LayoutItemBackgroundPointerOverBrush" Color="#FF333333" />
+    <SolidColorBrush x:Key="LayoutPreviewZoneBackgroundBrush" Color="#BF1b1b1b" />
+    <SolidColorBrush x:Key="LayoutPreviewBackgroundBrush" Color="#331B1B1B" />
+    <SolidColorBrush x:Key="LayoutPreviewZoneBorderBrush" Color="#FF1b1b1b" />
```

Comment:
Should be the same LayoutItemPreviewZoneBorderBrush as defined in Dark.xaml

Reply:
We need to add a little bit of contrast, right now the zones in both light and dark themes are difficult to see.

Reply:
Updated, now these colors are the same as in Dark.xml.

Reply:
We were using the same colors as the updated Alarms & Clock app, but I agree.. contrast is pretty low. The WinUI / Design have some outstanding PRs related to Design Tokens that include updated background colors. We can move to those once they are finalized.